modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/reducers/segmentsExperiments.es.js

```diff
@@ -29,7 +29,7 @@ export function createSegmentsExperimentsReducer(state, action) {
 function _createExperiment({name, description, segmentsExperienceId}) {
 	return new Promise(function(resolve, reject) {
 		Liferay.Service(
-			'/segments.segmentsexperiment/add-segments-experience',
+			'/segments.segmentsexperiment/add-segments-experiment',
```

Comment:
😰 @andresfulla

Reply:
ouch!
also, is `/segments.segmentsexperiment` fine in singular?